server.js

```diff
@@ -5,10 +5,10 @@ process.env.PORT = '3000';
 
 app.use(express.static(__dirname + '/dist'));
 
-app.get('*', function(req, res){
+app.get('*', function (req, res) {
   res.sendfile(__dirname + '/dist/index.html');
 });
 
 app.listen(process.env.PORT, () => {
```

Comment:
Увидела строчку `process.env.PORT = '3000';`
Переменные окружения берутся из env файла или аргументов команды, в коде не устанавливаются
```suggestion
app.listen(process.env.PORT ?? 3000, () => {
```